packages/libtool.rb

```diff
@@ -8,22 +8,22 @@ class Libtool < Package
   source_sha256 'e3bd4d5d3d025a36c21dd6af7ea818a2afcd4dfc1ea5a17b39d7854bcd0c06e3'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-x86_64.tar.xz',
+#    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
+#     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-armv7l.tar.xz',
+#       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-i686.tar.xz',
+#     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtool-2.4.6-2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '8c590c1a92eb3fba9a341a503c5cdd1bcc0bc505a261c43ce3a980aac94ad18d',
-     armv7l: '8c590c1a92eb3fba9a341a503c5cdd1bcc0bc505a261c43ce3a980aac94ad18d',
-       i686: 'ab153de8259f9eccd2ffbe3ef6eac2bad7d483d6338b064332abae11a3d00ac8',
-     x86_64: 'aebbf436953ab2ed8d844ec2449216578243fda5145c5cdb12c257f388e863c2',
+#    aarch64: '8c590c1a92eb3fba9a341a503c5cdd1bcc0bc505a261c43ce3a980aac94ad18d',
+#     armv7l: '8c590c1a92eb3fba9a341a503c5cdd1bcc0bc505a261c43ce3a980aac94ad18d',
+#       i686: 'ab153de8259f9eccd2ffbe3ef6eac2bad7d483d6338b064332abae11a3d00ac8',
+#     x86_64: 'aebbf436953ab2ed8d844ec2449216578243fda5145c5cdb12c257f388e863c2',
```

Comment:
Please remove the lines instead of commenting them.

Reply:
Corrected in commit 495c345